### DISCUSSION TITLE
translate student plan?

### DISCUSSION THREAD

**Discussion by pedromarquetti** [community/community#67866] [GitHub Education]
Hi. How are y'all doing?

I want to enroll in the student program, I do have an university email, but my university is remote, so I don't have an id to show at the proof of academic status. 
I do have a signed and dated letter in Portuguese proving I am enrolled in the course, but, should I translate it? Is it valid?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @pedromarquetti , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by pedromarquetti** [reply]
Oh. Thanks.  Didn't know how to properly tag my comment
